### ISSUE TITLE
Database has no foreign key

### ISSUE THREAD

**Issue by xavier-GitHub76** [gotify/server#667]
Hello,

Database has no foreign key : 
![image](https://github.com/user-attachments/assets/0fe1d789-04f9-4fcc-9f4e-e1c19cdcbca3)

It could be possible to insert bad value on referenced key.

The attached script add foreign keys.
[Foreign Keys for gotify.zip](https://github.com/user-attachments/files/16267329/Foreign.Keys.for.gotify.zip)

After, the database structure will be : 
![image](https://github.com/user-attachments/assets/bf1d3589-ff30-4f8f-aeee-457620229f03)

Best regards

**Comment by jmattheis** [comment]
I agree it would be more robust when there are foreign keys, but this is more of an improvement than a bug.

**Comment by xavier-GitHub76** [comment]
I agree with you. Thanks for your work.
